addons/intel/e810.go

```diff
@@ -89,15 +90,21 @@ func OnPTPConfigChangeE810(data *interface{}, nodeProfile *ptpv1.PtpProfile) err
 				glog.Error("e810 failed to unmarshal opts: " + err.Error())
 			}
 			if e810Opts.EnableDefaultConfig {
-				stdout, err = exec.Command("/usr/bin/bash", "-c", EnableE810PTPConfig).Output()
+				stdout, _ = exec.Command("/usr/bin/bash", "-c", EnableE810PTPConfig).Output()
 				glog.Infof(string(stdout))
 			}
 			if (*nodeProfile).PtpSettings == nil {
 				(*nodeProfile).PtpSettings = make(map[string]string)
 			}
 			for device, pins := range e810Opts.DevicePins {
 				dpllClockIdStr := fmt.Sprintf("%s[%s]", dpll.ClockIdStr, device)
-				(*nodeProfile).PtpSettings[dpllClockIdStr] = strconv.FormatUint(getClockIdE810(device), 10)
+				if _, found := (*nodeProfile).PtpSettings["unitTest"]; found {
+					buf := make([]byte, 8)
+					rand.Read(buf)
+					(*nodeProfile).PtpSettings[dpllClockIdStr] = strconv.FormatUint(binary.LittleEndian.Uint64(buf), 10)
```

Comment:
This code is to assign randomly-generated uuid?  Can you move this to a generate_uuid function?

Reply:
No, this is not a UUID. ClockID is a UINT64 number. UUID package either gives a string, or a 16-byte buffer, that I would need to cut and turn to UINT64 anyway. Not sure this will simplify the code. WDYT?